runtime/src/bank.rs

```diff
@@ -455,6 +455,9 @@ pub struct Bank {
     pub operating_mode: Option<OperatingMode>,
 
     pub lazy_rent_collection: AtomicBool,
+
+    // this is temporary field only to remove rewards_pool entirely
+    pub rewards_pool_pubkeys: Arc<HashSet<Pubkey>>,
```

Comment:
We need to serde skip this right?

Reply:
@mvines Well, not anymore. Bank isn't `derive(Serialize)` anymore after #10581. `derive(Serialize)` is moved to versionized actual `struct`s for each snapshot version.

Reply:
nice!